base/test-cargo.bats

```diff
@@ -30,9 +30,13 @@ testHelloCargo() {
   assert_success
   run xx-cargo build --verbose --color=never --manifest-path=./fixtures/hello_cargo/Cargo.toml --release --target-dir /tmp/cargobuild
   assert_success
-  xx-verify /tmp/cargobuild/$(xx-cargo --print-target)/release/hello_cargo
+  sfx=""
+  if [ "$TARGETARCH" = "wasm" ]; then
+    sfx=".wasm"
```

Comment:
Don't like this hack but I can't see a way how to configure this in cargo. Maybe `xx-cargo` should scan for the target dir and create hardlink/symlink to the correct path?